File: Section Reports/Layout/Stylesheets/C#/Stylesheets/ProductsReport.cs
namespace GrapeCity.ActiveReports.Samples.StyleSheets
{
	partial class ProductsReport : GrapeCity.ActiveReports.SectionReport
	{
		public ProductsReport()
		{
			InitializeComponent();
			Data.OleDBDataSource dS = new Data.OleDBDataSource();
			dS.ConnectionString = Properties.Resources.ConnectionString;
			dS.SQL = "Select Products.ProductName, Products.UnitPrice, Products.CategoryID, categories.CategoryName FROM Products, Categories WHERE Products.CategoryID = Categories.CategoryID Order By Categories.CategoryID;";
			DataSource = dS;
		}
	}
}

File: Section Reports/Layout/Stylesheets/VB.NET/Stylesheets/ProductsReport.vb
Public Class ProductsReport 
	Public Sub New()
		InitializeComponent()
		Dim ds As New Data.OleDBDataSource
		ds.ConnectionString = My.Resources.ConnectionString
		ds.SQL = "Select Products.ProductName, Products.UnitPrice, Products.CategoryID, categories.CategoryName FROM Products, Categories WHERE Products.CategoryID = Categories.CategoryID Order By Categories.CategoryID;"
		DataSource = ds
	End Sub
End Class
